src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -254,28 +253,54 @@ def constant(
                           memory is reflected on this Constant's memory!
                           If `False`, data is being copied to this Constant.
                           Requires data to be C_CONTIGUOUS if `True`.
+                          Disabled by default if:
+                          - value is a scalar.
+                          - dtype is one of: Type.u1, Type.i4, Type.u4, Type.nf4, Type.bf16.
+                          - dtype force conversion of data.
     :return: The Constant node initialized with provided data.
     """
+
+    # def __warning_message(msg, shared_memory):
```

Comment:
Remove not used code?

Reply:
For some reason I was able to see it as well but it was removed with [2f5b096](https://github.com/openvinotoolkit/openvino/pull/21046/commits/2f5b09627d44c521ce240967b1f391757bd68301)